templates/contactList.html

```diff
@@ -1,6 +1,6 @@
 <div class="contacts-list" ng-class="{loading: ctrl.loading, 'mobile-show': ctrl.show}">
         <div class="app-content-list-item"
-			 ng-repeat="contact in ctrl.filteredContacts = (ctrl.contactList | contactGroupFilter:ctrl.routeParams.gid | localeOrderBy:ctrl.sortBy | filter:query | limitTo:ctrl.limitTo ) as filtered track by contact.uid()"
+			 ng-repeat="contact in ctrl.filteredContacts = (ctrl.contactList | filter:{'data':{'addressBook':{'enabled': true}}} | contactGroupFilter:ctrl.routeParams.gid | localeOrderBy:ctrl.sortBy | filter:query | limitTo:ctrl.limitTo) as filtered track by contact.uid()"
```

Comment:
For some reason it doesnt work! :/
@Henni ?

Reply:
As far as I can see, you only store the state in localstorage and don't update it inside the addressbook itself.
This update is only applied on page reload (https://github.com/nextcloud/contacts/pull/412/files#diff-adfefcaa5c468a1b21ca840b249cadcfR14).

Why don't you store it on the server in the first place?

Reply:
I don't think we can.

Reply:
Should be possible, calendar does it. But they use a property in the 'owncloud' namespace.
```
<d:propertyupdate xmlns:d="DAV:"><d:set><d:prop><o:calendar-enabled xmlns:o="http://owncloud.org/ns">0</o:calendar-enabled></d:prop></d:set></d:propertyupdate>
```
@irgendwie Is such a thing possible with our current dav lib?

Reply:
Oooh yeah, I remember this! I don't see where it's stored in the db though! :/

Reply:
IIRC, the settings weren't saved.

Reply:
Okay, I needed a patch for the dav lib.
Easy fix now! :)

All good :)